app/locale/en.coffee

```diff
@@ -1781,7 +1781,7 @@
     gd3_curriculum_desc: "Lesson plans for Game Development 3."
     wd1: "Web Development 1"
     wd1_curriculum: "Web Development 1 - Curriculum Guide"
-    wd1_curriculum_desc: "Lesson plans for Web Development 1."
+    wd1_curriculum_desc: "Scope and sequence, lesson plans, activities, and more for Web Development 1."
```

Comment:
Need to add `# {change} to the end of this line to mark it for translation to other locales.

Reply:
Thank you for catching this! 🙏